charts/fybrik/values.yaml

```diff
@@ -171,6 +171,8 @@ manager:
 
   prometheus: false
 
+  chartVolume: ""
```

Comment:
Consider adding a comment on how to use it.

Reply:
Done